### ISSUE TITLE
Add Support for replacement in CommentTag

### ISSUE THREAD

**Issue by harboe** [krisztianb/typedoc-plugin-replace-in-comments#1]
Hi Krisztián,

I'm trying to make our `typedoc` code examples testable and I came by you're plugin. can see in the code they isn't any replacement in the `CommentTag` is this by intension? or nobody had this requirement? So here is the proposed changes, After line:
https://github.com/krisztianb/typedoc-plugin-replace-in-comments/blob/68edce23d37587698ec9dc63dac8cb3ed0732f1a/src/plugin.ts#L61

Add:
```ts
reflection.comment.tags = reflection.comment.tags.map((t) => {
    return {...t, text: this.replaceInComment(t.text) };
}); 
```
It would be awesome. 😄 

May thanks.
Dennis

**Comment by krisztianb** [comment]
Hi Dennis. For some reason nobody needed this yet. But it makes sense, so I accept the feature request. Thanks.

**Comment by krisztianb** [comment]
Released with 3.2.0 🎉
